src/set_theory/game/basic.lean

```diff
@@ -185,45 +185,65 @@ def to_right_moves_mul {x y : pgame} :
-  (mk xl xr xL xR * mk yl yr yL yR).move_left (sum.inr (i, j))
-  = xR i * (mk yl yr yL yR) + (mk xl xr xL xR) * yR j - xR i * yR j :=
+  (mk xl xr xL xR * mk yl yr yL yR).move_left (sum.inr (i, j)) =
+  xR i * mk yl yr yL yR + mk xl xr xL xR * yR j - xR i * yR j :=
 rfl
 
 @[simp] lemma mul_move_left_inr {x y : pgame} {i j} :
-   (x * y).move_left (to_left_moves_mul (sum.inr (i, j)))
-   = x.move_right i * y + x * y.move_right j - x.move_right i * y.move_right j :=
+  (x * y).move_left (to_left_moves_mul (sum.inr (i, j))) =
+  x.move_right i * y + x * y.move_right j - x.move_right i * y.move_right j :=
 by { cases x, cases y, refl }
 
 @[simp] lemma mk_mul_move_right_inl {xl xr yl yr} {xL xR yL yR} {i j} :
-  (mk xl xr xL xR * mk yl yr yL yR).move_right (sum.inl (i, j))
-  = xL i * (mk yl yr yL yR) + (mk xl xr xL xR) * yR j - xL i * yR j :=
+  (mk xl xr xL xR * mk yl yr yL yR).move_right (sum.inl (i, j)) =
+  xL i * mk yl yr yL yR + mk xl xr xL xR * yR j - xL i * yR j :=
 rfl
 
 @[simp] lemma mul_move_right_inl {x y : pgame} {i j} :
-   (x * y).move_right (to_right_moves_mul (sum.inl (i, j)))
-   = x.move_left i * y + x * y.move_right j - x.move_left i * y.move_right j :=
+  (x * y).move_right (to_right_moves_mul (sum.inl (i, j))) =
+  x.move_left i * y + x * y.move_right j - x.move_left i * y.move_right j :=
 by { cases x, cases y, refl }
 
 @[simp] lemma mk_mul_move_right_inr {xl xr yl yr} {xL xR yL yR} {i j} :
-  (mk xl xr xL xR * mk yl yr yL yR).move_right (sum.inr (i, j))
-  = xR i * (mk yl yr yL yR) + (mk xl xr xL xR) * yL j - xR i * yL j :=
+  (mk xl xr xL xR * mk yl yr yL yR).move_right (sum.inr (i, j)) =
+  xR i * mk yl yr yL yR + mk xl xr xL xR * yL j - xR i * yL j :=
 rfl
 
 @[simp] lemma mul_move_right_inr {x y : pgame} {i j} :
-   (x * y).move_right (to_right_moves_mul (sum.inr (i, j)))
-   = x.move_right i * y + x * y.move_left j - x.move_right i * y.move_left j :=
+  (x * y).move_right (to_right_moves_mul (sum.inr (i, j))) =
+  x.move_right i * y + x * y.move_left j - x.move_right i * y.move_left j :=
 by { cases x, cases y, refl }
 
+@[simp] lemma mk_neg_mul_move_left_inl {xl xr yl yr} {xL xR yL yR} {i j} :
+  (- (mk xl xr xL xR * mk yl yr yL yR)).move_left (sum.inl (i, j)) =
+  - (xL i * mk yl yr yL yR + mk xl xr xL xR * yR j - xL i * yR j) :=
+rfl
+
+@[simp] lemma mk_neg_mul_move_left_inr {xl xr yl yr} {xL xR yL yR} {i j} :
+  (- (mk xl xr xL xR * mk yl yr yL yR)).move_left (sum.inr (i, j)) =
+  - (xR i * mk yl yr yL yR + mk xl xr xL xR * yL j - xR i * yL j) :=
+rfl
+
+@[simp] lemma mk_neg_mul_move_right_inl {xl xr yl yr} {xL xR yL yR} {i j} :
+  (- (mk xl xr xL xR * mk yl yr yL yR)).move_right (sum.inl (i, j)) =
+  - (xL i * mk yl yr yL yR + mk xl xr xL xR * yL j - xL i * yL j) :=
+rfl
+
+@[simp] lemma mk_neg_mul_move_right_inr {xl xr yl yr} {xL xR yL yR} {i j} :
+  (- (mk xl xr xL xR * mk yl yr yL yR)).move_right (sum.inr (i, j)) =
+  - (xR i * mk yl yr yL yR + mk xl xr xL xR * yR j - xR i * yR j) :=
+rfl
+
```

Comment:
These are not mentioned in the PR description nor used anywhere; look like a scope creep. And I don't think we ever add a space after negation `-`.

Reply:
They are implicitly used in the proof of `mul_neg_relabelling` as `dsimp` lemmas. I'll mention them in the PR description.

Reply:
Sure thing.